src/content/developers/docs/networking-layer/portal-network/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: The Portal Network
+description: An overview of the Portal Network - an in-development network designed to support low-resource clients.
+lang: en
+---
+
+Ethereum runs on nodes. A "node" is any instance of Ethereum client software connected to other computers also running Ethereum software, forming a network. A client is an implementation of Ethereum that verifies data against the protocol rules and keeps the network secure.
```

Comment:
We don't want to repeat the basics of what are nodes etc too much here imo. Let's tighten this up and get right to the point of the page.

```suggestion
Ethereum is a network made up of computers that run Ethereum client software. Each of these computers is called a 'node'. The client software allows a node to send and receive data on the Ethereum network, and verifies data against the Ethereum protocol rules. Nodes keep a lot of historical data in their disk storage and add to it when they receive new packets of information, known as blocks, from other nodes on the network. This is necessary for always checking that a node has information consistent with the rest of the network. This means running a node can require a lot of disk space. Some node operations can require a lot of RAM too.

To get around this disk storage problem, 'light' nodes have been developed that request information from full nodes instead of storing it all themselves. However, this means the light node is not independently verifying the information and is trusting another node instead. It also means that full nodes are required to take on extra work to serve those light nodes.

The Portal network is a new networking design for Ethereum that aims to solve the data availability problem for "light" nodes without having to trust or put extra strain on full nodes, by sharing the necessary data in small chunks across the network.

Read more about [nodes and clients]((https://ethereum.org/en/developers/docs/nodes-and-clients/))
```